docs/specification.md

```diff
@@ -90,9 +92,12 @@ Required `Holdings` entrypoints:
 * `setSafelistAddress ((option address) :mbNewSafelistAddress)`
   * Description: updates `safelistAddress` field, which is used for safelist contract calls.
   * Constraints: `onlyOwner`.
-* `setAdminAddress (address)`
-  * Description: updates `admin` field.
+* `transferAdminRights (address :newAdmin)`
+  * Description: transfers `admin` rights to a new address.
```

Comment:
I would add that this transfer is not complete until the new admin calls `acceptAdminRights`.